teuthology/task/install.py

```diff
@@ -577,6 +529,23 @@ def remove_sources(ctx, config):
             p.spawn(remove_fn, remote, project)
 
 
+def get_package_list(config):
+    debug = config.get('debuginfo', False)
+    project = config.get('project', 'ceph')
+    yaml_path = os.path.join(os.path.dirname(__file__), 'packages.yaml')
```

Comment:
teuthology doesn't currently depend on ceph-qa-suite, and what you're proposing would change that.